sql_composer/pg/pg_translator.py

```
@@ -0,0 +1,585 @@
+            PgFilterOp.FULLTEXT_QUERY,
+            PgFilterOp.IS_DISTINCT_FROM,
+            PgFilterOp.IS_NOT_DISTINCT_FROM,
+            PgFilterOp.ANY,
+            PgFilterOp.ALL,
+            PgFilterOp.SOME,
+            PgFilterOp.EXISTS,
+            PgFilterOp.NOT_EXISTS,
+        ):
+            raise ValueError(
+                f"Operator {where.op} requires exactly 1 value, got {len(where.values)}"
+            )
```

Ruff (0.14.7)

Avoid specifying long messages outside the exception class

(TRY003)